src/assets/sass/variables/colors.scss

```
@@ -22,6 +22,7 @@ $light-blue-700: #3e7e9b;
 $light-blue-800: #306178;
 $light-blue-900: #254a5c;
 
+$blue-25: #e5ecff;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)